dojo/tools/mobsf/parser.py

```diff
@@ -139,6 +139,25 @@ def get_findings(self, filename, test):
                                 "file_path": details["name"]
                             }
                             mobsf_findings.append(mobsf_item)
+            elif data["binary_analysis"].get("findings"):
+                for binary_analysis_type, details in list(data["binary_analysis"]["findings"].items()):
+                    # "findings":{
+                    #     "Binary makes use of insecure API(s)":{
+                    #         "detailed_desc":"The binary may contain the following insecure API(s) _memcpy\n, _strlen\n",
+                    #         "severity":"high",
+                    #         "cvss":6,
+                    #         "cwe":"CWE-676: Use of Potentially Dangerous Function",
+                    #         "owasp-mobile":"M7: Client Code Quality",
+                    #         "masvs":"MSTG-CODE-8"
+                    #     },
+                    mobsf_item = {
+                        "category": "Binary Analysis",
+                        "title": details["detailed_desc"],
```

Comment:
Seems like the `binary_analysis_type` might serve as a better title here, at least based on the examples included below?

Reply:
I just used here the codepart from below (line 162) which worked out of the box. I think this is fine